homeassistant/components/camera/arlo.py

```diff
@@ -74,31 +72,41 @@ def __init__(self, hass, camera, device_info):
         self._ffmpeg = hass.data[DATA_FFMPEG]
         self._ffmpeg_arguments = device_info.get(CONF_FFMPEG_ARGUMENTS)
         self._last_refresh = None
-        if self._camera.base_station:
-            self._camera.base_station.refresh_rate = \
-                SCAN_INTERVAL.total_seconds()
         self.attrs = {}
 
     def camera_image(self):
         """Return a still image response from the camera."""
-        return self._camera.last_image
+        return self._camera.last_image_from_cache
+
+    async def async_added_to_hass(self):
+        """Register callbacks."""
+        async_dispatcher_connect(
+            self.hass, SIGNAL_UPDATE_ARLO, self._update_callback)
+
+    @callback
+    def _update_callback(self):
+        """Call update method."""
+        self.async_schedule_update_ha_state(True)
```

Comment:
There's no `update` method for this entity anymore so we don't need the argument `True` here. Sorry if I said so earlier.